src/diff/children.js

```diff
@@ -24,7 +24,7 @@ import { removeNode } from '../util';
  * Fragments that have siblings. In most cases, it starts out as `oldChildren[0]._dom`.
  */
 export function diffChildren(parentDom, newParentVNode, oldParentVNode, context, isSvg, excessDomChildren, mounts, ancestorComponent, oldDom) {
-	let childVNode, i, j, oldVNode, newDom, sibDom;
+	let childVNode, i, j, oldVNode, newDom, sibDom, lastRendered;
```

Comment:
can't we default this to childVNode, saves some bytes since we can remove line 88 like that